cogs/vxtwitconverter/helpers.py

```diff
@@ -1,5 +1,33 @@
+import re
+
 from discord import Embed, Message, channel
 
+from .constants import SocialMedia
+
+
+def convert_to_ddinsta_url(embeds: list[Embed]):
+    """
+    Parameters
+    ----------
+    embeds: list of discord embeds
+
+    Returns
+    -------
+        filtered list of Instagram URLs that have been converted to ddinstagram
+    """
+
+    # pulls only video embeds from list of embeds
+    urls = [entry.url for entry in embeds]
+
+    INSTA_REGEX_MATCH = r"https://(?:www\.)?(instagram.com)"
```

Comment:
Can we have this as a `re.Pattern` constant just so it's compiled upon cog import?